### PR TITLE
Darren bug fix

### PR DESCRIPTION
**Pull request by medzernik**
Fix for the Darren's multiple key trigger not executing correct text and multiple macros on single trigger bug

### CODE REVIEW

**Comment by medzernik** [comment]
Will be closing #75

**Comment by medzernik** [comment]
Will be closing #87 -- at least everything we can fix.

**Comment by medzernik** [comment]
Closing #80

File: wooting-macro-backend/src/lib.rs [hunk 1]
```diff
@@ -319,13 +319,15 @@ fn check_macro_execution_efficiently(
...
 
-                            // ? Kinda works for now but needs to be improved. Disabled for now as its more of a regression than a fix.
+                            let channel_clone_lift = channel_sender.clone();
+                            let channel_clone_execute = channel_sender.clone();
+                            let macro_clone_execute = macros.clone();
+                            let pressed_data_clone = data.clone();
 
-                            let channel_clone = channel_sender.clone();
-                            let macro_clone = macros.clone();
+                            plugin::util::lift_keys(pressed_data_clone, channel_clone_lift);
```
Comment (simon-wh) [inline wooting-macro-backend/src/lib.rs:328]: Do these really need to be cloned to be given to lift_keys? I think now you should just be able to pass them by reference to be used briefly

**Review by simon-wh — APPROVED** [review]

**Pull request merged**